test/integration/smoke.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { test, expect, Request } from '@playwright/test';
+import { test, expect, Request } from '../playwright/test';
 import generateId from '../utils/generateId';
 import * as locators from '../utils/locators';
 
-test('basic app creation flow', async ({ page }) => {
+test.only('basic app creation flow', async ({ page }) => {
```

Comment:
we should have an eslint rule for these if we don't, i missed it before in my PRs too

Reply:
I added the `--forbid-only` flag in CI